test/e2e/globals.js

```diff
@@ -1,5 +1,5 @@
 export default {
-  waitForConditionTimeout: 5000,
+  waitForConditionTimeout: 8000,
```

Comment:
but it's still better than running whole test twice, isn't it? i'm not a big fan too, but it just works.